### ISSUE TITLE
[Suggestion] Setup for i18n so translator could helps

### ISSUE THREAD

**Issue by maxlandry** [bitwarden/web#46]
I know I talked to you about this on gitter @kspearrin, but I also want to write it here to make it more formal.

I think it would be great for the webvault to be i18n ready, so translators could help you localize the platform!

Once it's done, I could help with the French language.

Thank you and keep up the good work!

**Comment by kspearrin** [comment]
The web vault is still going through a lot of development right now which means that strings are changing constantly. Once things die down a bit and a lot of the language is finalized we'll definitely be looking to make it i18n ready.

**Comment by bcbane** [comment]
If translation ever becomes something actively pursued, I would be happy to help with Spanish translation

**Comment by kspearrin** [comment]
@bcbane We need Spanish on the browser extension and mobile if you are interested. They are available for translations today.

**Comment by EMLVIRUS** [comment]
I can help with Zh-CN

**Comment by a-polivanchuk** [comment]
Hi,
I'm also waiting to help with Ukrainian translation.
Please let us know in this thread when the web will be available for localization.

**Comment by turbomettwurst** [comment]
Is this still being worked on? 
I found https://crowdin.com/project/bitwarden-web but it has nothing to translate?

**Comment by kspearrin** [comment]
@turbomettwurst Yes, translations will be published to that project in the coming weeks. Stay tuned
